nifi-registry-framework/src/main/resources/db/migration/V1__Initial.sql

```diff
@@ -43,4 +43,11 @@ CREATE TABLE FLOW_SNAPSHOT (
     COMMENTS VARCHAR(4096),
     PRIMARY KEY (FLOW_ID, VERSION),
     FOREIGN KEY (FLOW_ID) REFERENCES FLOW(ID)
+);
+
+CREATE TABLE SIGNING_KEY (
+    ID VARCHAR2(50) NOT NULL,
+    TENANT_IDENTITY VARCHAR2(50) NOT NULL UNIQUE,
+    KEY_VALUE VARCHAR2(50) NOT NULL,
+    PRIMARY KEY (ID)
```

Comment:
Let me know if the preference is to make this V2__*.sql. I added it here assuming that we didn't have to worry about any production deployments of nifi-registiry-0.0.1-SNAPSHOT 😄